content/about/myths.md

```diff
@@ -21,6 +21,36 @@ In "typical" parsers, XML and JSON perform more or less identically in practice.
 
 But even ignoring the high performance, specialist end, XML still holds its own. Consider [these findings](https://www.balisage.net/Proceedings/vol10/html/Lee01/BalisageVol10-Lee01.html), for example.
 
+## XMPP is not secure.
+
+The hypothesis:
+> XMPP is not secure.
+
+The fact:
+> XMPP offers state-of-the-art end-to-end encryption, and a relatively high-degree of pseudonymity
+
+"Secure" is not an absolute. In order to decide whether XMPP is secure, you need to decide what you want to be secure against. No system that relies on servers is secure against for example a rogue administrator (end-to-end encryption can protect your message contents even against a rogue admin, but only if you verify keys appropriately). Instead, our specifications consistently remind developers of the risks of various choices. Plenty of XMPP extensions will certainly increase the risk for both for the users and for the administrator - including allowing registration and password changing via the protocol.
```

Comment:
> Instead, our specifications consistently remind developers of the risks of various choices.

Might want to specifically mention the `Security Considerations` section.

For example:

Instead, XMPP specifications usually contain a "Security Considerations" section where developers are made aware of the risks of various choices.